packages/vite/src/utils/executor-utils.ts

```diff
@@ -43,15 +42,7 @@ export function registerPaths(
       context.targetName,
       context.configurationName
     );
-    const tmpTsConfig = createTmpTsConfig(
-      tsConfig,
-      context.root,
-      projectRoot,
-      dependencies
-    );
-
-    registerTsConfigPaths(tmpTsConfig);
-  } else {
-    registerTsConfigPaths(tsConfig);
```

Comment:
As far as I can tell, since these paths never being used by vite downstream, there isn't a reason to register them anymore.